### DISCUSSION TITLE
Cann't pip install deepspeed==0.11.1

### DISCUSSION THREAD

**Discussion by nothing7744** [deepspeedai/DeepSpeed#6535] [Q&A] [answered]
Since I want to reproduce the project, I need to download deepspeed 0.11.1, but I can't download it via pip command and it reports the following error.
![bug](https://github.com/user-attachments/assets/c599e264-d244-487c-9ba8-b10006a628db)

**Comment by loadams** [comment, marked as answer]
This looks to be a timeout issue in your download, you can try with a better network connection or with an increased timeout being passed to the pip command.  However, this DeepSpeed version is quite old and we recommend using a newer one.
